### DISCUSSION TITLE
Translation questions

### DISCUSSION THREAD

**Discussion by Gapstare** [tenacityteam/tenacity-legacy#313] [Q&A]
I just started working on the not-there-yet translation to Swedish (that also have some other issues). At what rate will the work done on Weblate be incorporated in the source files here on GitHub?

**Comment by fossdd** [comment]
@emabrey

**Comment by emabrey** [comment]
https://github.com/tenacityteam/tenacity/pull/323

**Reply by Gapstare** [reply]
Thanks @emabrey 

This is not a top priority by any means, but when this project settles down a bit more translation hints could be used. Depending on context different words and grammatical constructions should be used and I imagine it differs wildly between languages.

I think I've seen strings like `%s, %s` or similar and those are of course impossible to translate. I wonder if there is a way to mark such strings as untranslatable?

**Comment by Gapstare** [comment]
More questions; 

- `Audacity` is hardcoded all over the .po-files, should the translator fix this or is it something that should be done _en masse_ by devs?
- Is there a way to mark a string as _wont translate_ for personal names and such? You can skip strings but they pop up again next time you log in to Weblate.

**Reply by nbsp** [reply]
1. Is it? Looking around in various languages on Weblate, the English text to translate *from* mentions Tenacity. If you're referring to your translation, please use Tenacity in all contexts (and don't transliterate it to your language!)
2. Ditto for crash reports, which it doesn't make sense to translate. @emabrey?

Next time, please send an email to `~tenacity/tenacity-discuss@lists.sr.ht` instead of this tab -- I, personally, rarely look in this Discussions tab.

If you'd like a real-time discussion instead of the forum format, `#tenacity-dev` on irc.libera.chat is the place for dev/i18n discussion.

**Reply by Gapstare** [reply]
- I'm blind to everything but the Swedish file so I probably shouldn't have written _files_. Anyway, in `sv.po` there is a bunch of mentions of Audacity. I don't know if this is a relic from past translators, but probably not - I just look on strings that haven't been translated yet for now. I promise I won't translate the name of the program, that rarely makes sense 😄 _"Are you using the Tenacity audio editor? No, I'm using Envishet!"_
- Do you mean that I shouldn't touch the strings related to crash reporting since they wont be used? Yeah, that's probably a waste of time. I just wish I didn't have to click skip on them every time, but I'll survive 🙂

**Reply by nbsp** [reply]
Translating crash logs is dumb because the ones reading the crash logs will rarely know the language it was translated to.

**Reply by poyribn** [reply]
Crashlogs are for the developers, not for the users.

Every developer today knows English by necessity because every widely-used programming language is based on English.

And even if English is not their first language, they can use translation tools like [Bing Translator](https://www.bing.com/translator) to help them understand the crashlog.
And even if that doesn't work, they can ask other developers for help on how to understand the crashlog, and the [grammatical error acceptance](https://i.kym-cdn.com/photos/images/original/002/153/234/ea4.jpg) in today's English-language culture makes it much less scary.

**Comment by Gregaras** [comment]
Can people still contribute to translations?
